### PR TITLE
Skip to links

### PR DESCRIPTION
**Pull request by radinamatic**
This is still a work-in-progress, so I'm submitting a PR mostly for feedback & help. 

The best implementation of "skip to" links should be useful for both screen-reader & sighted keyboard navigation users. This solution is inspired by the one on the http://webaim.org/

Two things remain to be solved/added here:  

1. After skipping to a particular point, there should be a way to grab the focus of the element. One possible solution would be something like described here (still working on how to implement it):
https://www.bignerdranch.com/blog/web-accessibility-skip-navigation-links/  

2. For some reason the two links inside the `{% if is_logged_in %}` condition are not showing now, even though they did display correctly (once the user logged in) before I put them inside the `<div id="skipto">`. I'm not really sure why... :(

### CODE REVIEW

**Comment by MCGallaspy** [comment]
We haven't forgotten about this :)

**Comment by radinamatic** [comment]
Thanks, @MCGallaspy! 
Yes, this still needs work, so 0.15 is a good milestone...

**Comment by rtibbles** [comment]
For cleaner commit histories, doing rebase instead of merge can help!

File: kalite/distributed/templates/distributed/base.html [hunk 1]
```diff
@@ -185,6 +185,21 @@
             <div class="navbar kalite-navbar header-footer-bg" role="banner">
               <div class="container">
                   <div class="row">
+
```
Comment (rtibbles) [inline kalite/distributed/templates/distributed/base.html]: This needs to go into the relevant handlebars template, I believe, not here!

**Comment by rtibbles** [comment]
Stale contrib and merge conflict.

**Comment by rtibbles** [comment]
@radinamatic What's the status on this?

**Comment by radinamatic** [comment]
Mea culpa, this got tota11y forgotten... :(

One of the reasons is that my plan was to implement this together with site-wide visual-focus-outline strategy, and for that I'd like to confer with @jtamiace. How soon should it be done to be viable for 0.15?

**Comment by rtibbles** [comment]
Code freeze is end of next week.

**Comment by radinamatic** [comment]
Then we better bump it to 0.16, since I doubt we'll have it by then (I'm due to travel next week, fever permitting).

**Comment by rtibbles** [comment]
What's the prospectus for this one?

**Comment by rtibbles** [comment]
Bumpity bump bump?!

**Comment by rtibbles** [comment]
A bump bump?

**Comment by benjaoming** [comment]
Closing this as @radinamatic has indicated there won't be time to finalize it immediately.

**Pull request closed without merging**